test/NFT/StableUnitDAOogNFTVotes-test.ts

```diff
@@ -0,0 +1,107 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+describe("StableUnitDAO ogNFTVotes", () => {
+    it("should mint new NFT, send old to DEAD address", async () => {
+        const accounts = await ethers.getSigners();
+        const owner = accounts[0];
+
+        const NftMock = await ethers.getContractFactory("NftMock");
+        const nftMock = await NftMock.deploy("NFT mock", "mNFT");
+        const SuDAOogVotes = await ethers.getContractFactory("StableUnitDAOogNFTVotes");
+        const suDAOogVotes = await SuDAOogVotes.deploy(nftMock.address);
+                    
+        await nftMock.mint(owner.address);
+        await nftMock.setApprovalForAll(suDAOogVotes.address, true);
+
+        await suDAOogVotes.mint();
+
+        await expect(await nftMock.balanceOf("0x000000000000000000000000000000000000dEaD")).to.equal(1);
+        await expect(await suDAOogVotes.totalSupply()).to.equal(1);
+    });
+
+    it("Should be able to mint NFTs when paused", async function () {
+        const accounts = await ethers.getSigners();
+        const owner = accounts[0];
+
+        const NftMock = await ethers.getContractFactory("NftMock");
+        const nftMock = await NftMock.deploy("NFT mock", "mNFT");
+        const SuDAOogVotes = await ethers.getContractFactory("StableUnitDAOogNFTVotes");
+        const suDAOogVotes = await SuDAOogVotes.deploy(nftMock.address);
+
```

Comment:
I see the code repetition here, please encapsulate it into separate function and reuse via call to avoid code duplication. Also, webstorm from JetBrains helps to spot it and to refactor.